frontend/packages/client/src/components/Community/StrategyEditorModal/index.js

```diff
@@ -84,6 +60,26 @@ export default function StrategyEditorModal({
 
   // user selected strategy move to second step to enter information
   const setStrategy = (strategyName) => {
+    //
+    // STRATEGY CONFIGURATION
+    //
+    // If strategy selected is 'one-address-one-vote'
+    // then no more information is required(FE uses strategy configuration)
+    // modal should be closed and strategy should be ready to be added
+    if (strategyName === 'one-address-one-vote') {
+      const strategyConfig =
+        networkConfig.strategiesConfig['one-address-one-vote'];
+      onDone({
+        name: strategyName,
+        contract: {
+          name: strategyConfig.name,
+          addr: strategyConfig.addr,
+          publicPath: strategyConfig.publicPath,
+          threshold: '0',
+        },
+      });
+      return;
+    }
```

Comment:
I dont think this is needed because the entire `one-address-one-vote` strategy will be removed the BE

Reply:
Aww, think that got put back in after testing with Jackson's changes.

Reply:
Ah yes, missed this one.